recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ test:
     - test -f ${PREFIX}/lib/libx264.a          # [unix]
     - test -f ${PREFIX}/lib/libx264.dylib      # [osx]
     - test -f ${PREFIX}/lib/libx264.so         # [linux]
+    - test -f ${PREFIX}/lib/libx264.so.{{ soname }}  # linux
```

Comment:
You can download a tarball from https://anaconda.org/conda-forge/x264/files to see how .so files are named in osx. Hint: `.152.dylib`.